packages/slate-html-serializer/src/index.js

```diff
@@ -327,8 +329,8 @@ class Html {
     const { document } = value
     const elements = document.nodes.map(this.serializeNode).filter(el => el)
     if (options.render === false) return elements
-
-    const html = renderToStaticMarkup(<body>{elements}</body>)
+    // Converts List to Array to support IE 11.
```

Comment:
Could this and similar comments be updated to say `Compat: Converts List to Array for IE11 support` for consistency with other compatibility comments?